sample_apps_reinvent2021/README.md

```diff
@@ -0,0 +1,8 @@
+# Amazon Timestream sample applications
+## Getting started
+Below is an example of a fully functional sample application that shows how to get started with Amazon Timestream reinvent 2021 features. This application shows how to enable magnetic tier upserts on a table, create a scheduled query and interpret its run, ingestion of multi measure records, and how to query on multi measure records. It is available for the following programming languages:
+
+* [Java](https://github.com/awslabs/amazon-timestream-tools/blob/master/sample_apps_reinvent2021/java/)
+* [Java v2](https://github.com/awslabs/amazon-timestream-tools/blob/master/sample_apps_reinvent2021/javaV2/)
```

Comment:
These links aren't correct, e.g. the Java v2 link should look like:
>https://github.com/awslabs/amazon-timestream-tools/tree/mainline/sample_apps_reinvent2021/javaV2

Reply:
It will redirect automatically to path you mentioned.   Copied this from existing readme.   try this example: https://github.com/awslabs/amazon-timestream-tools/blob/master/sample_apps/java/

Reply:
Works, nice. Resolving this conversation.